keyboards/geonworks/frogmini/fmh/rules.mk

```diff
@@ -0,0 +1,25 @@
+MCU = STM32F401
+
```

Comment:
```suggestion
BOARD = GENERIC_STM32_F401XC

```

Reply:
Committed in [467b990](https://github.com/qmk/qmk_firmware/pull/17765/commits/467b990b90bb8d4de64eebe48136352a12dfaf20)